src/Filters/WhereIdIn.php

```diff
@@ -111,16 +107,6 @@ public function isSingular(): bool
         return false;
     }
 
-    /**
-     * Get the column for the ID.
-     *
-     * @return string|null
-     */
-    protected function column(): ?string
```

Comment:
The column method here is slightly different - it's nullable. Please can you restore this class (i.e. remove the trait and leave the nullable column implementation).

FYI the reason this one is nullable is because it is able to fall back to `Model::getKeyName()`.

Reply:
Why wouldn't you use the same pattern as the other filters? When column is not set guess the column based on the keyname? The logic for the guesscolumn is not part of the trait.

There is an inconsistency anyways where the contructor does not allow to specify the column in the WhereIdIn. 

Am i missing something there?

Reply:
This is because throughout the package, the fallback for the resource `id` column is *always* `Model::getRouteKey()`. So there's no need to guess the column name in this filter like the other filters - and the `column()` method needs to return `null` to indicate that the fallback should be used. `Model::getRouteKey()` can't be called until the query is passed to the filter, as it has to get the model from the Laravel query object.

So this definitely does need reverting in this filter.

Reply:
Ok, give me a sec. ;)